src/ssl.c

```diff
@@ -30357,6 +30371,8 @@ static int set_curves_list(WOLFSSL* ssl, WOLFSSL_CTX *ctx, const char* names)
     int *groups;
 #else
     int groups[WOLFSSL_MAX_GROUP_COUNT];
+
+    XMEMSET(groups, 0, sizeof(groups));
```

Comment:
This is another expensive `memset` to address a clearly-false positive, as above. in `wolfSSL_CTX_set1_groups()` etc.